lib/util/dateDiff.ts

```diff
@@ -0,0 +1,32 @@
+export const dateDiff = (createdDate: Date | undefined, nowDate: Date | undefined): string | undefined => {
```

Comment:
타입 스크립트를 사용할때 파라미터의 값이 옵셔널하다면 작성하신것처럼 `createdDate: Date | undefined` 와 같이 작성하지 않습니다.
타입스크립트에는 옵셔널 파라미터가 존재해 아래와 같이 수정할 수 있습니다.

```suggestion
export const dateDiff = (createdDate: Date | undefined, nowDate: Date | undefined): string | undefined => {
export const dateDiff = (createdDate?: Date, nowDate?: Date): string => {
```